app/util/autoversioning.py

```diff
@@ -20,18 +20,30 @@ def get_version():
     :return: The version of the application
     :rtype: str
     """
-    if getattr(sys, 'frozen', False):
-        return _get_frozen_package_version()  # frozen/packaged
 
-    return _calculate_source_version()  # unfrozen/running from source
+    # Initialize default version to '0.0.0'. So in case of failure the version will be shown as '0.0.0'.
+    version = '0.0.0'
```

Comment:
I think '0.0.0' should be a constant if we're using it as a comparison. Alternatively you can initialize version to `None` (both here and in package_version.py) and then do a `return version or '0.0.0'` at the end of this method.